department-of-veterans-affairs/caseflow
# Update Seed Data

Resolves [CASEFLOW-2792](https://vajira.max.gov/browse/CASEFLOW-2792) | Update Seed Data for Pending-Appeal Substitution

### Description
1. Updates existing seed data for appeal substitutions to include tasks. 
2. Adds two new sanitized appeals from prod to `/db/seeds/sanitized_json/`.

### Acceptance Criteria
- [ ] Substitution appeals seed data includes tasks.
- [ ] New sanitized appeals are present in `/db/seeds/sanitized_json/`.

### Testing Plan
Assumes you have a 'fresh' database, may have to run `make reset`.

PART 1
1. Search for case number '54545454'. 
2. Pull up a case that has not been dispatched.
3. Case should have a number of active tasks and completed tasks.
4. Next, pull up a case that has been dispatched. 
5. Check that there are active tasks present (should be two active tasks). 

PART 2
1. Open caseflow in your code editor.
2. Navigate to `/db/seeds/sanitized_json/`.
3. Check that `appeal-126187.json` and `appeal-119577.json` are in this folder.
